_posts/2021-04-01-moveit3_release.md

```diff
@@ -0,0 +1,39 @@
+---
+author: PickNik Team
+comments: false
+date: 2021-04-01
+layout: post
+title: MoveIt‼ 3 Release
+media_type: image
+media_link: /assets/images/blog_posts/moveit3.png
+description: Announcing the release of MoveIt‼ 3 for ROS 3!
+categories:
+- ROS
+- MoveIt
+---
+
+PickNik Robotics is proud to announce that MoveIt‼ 3 is being released for ROS 3! With ROS 2 facing EOL in just 163 months, the MoveIt‼ community has been hard at work porting the core functionality of MoveIt‼ to ROS 3, as well as introducing some exciting new features, including
+- Novel act-plan-sense control paradigm
```

Comment:
```suggestion
- Improve on slow real-time planning with the novel act-plan-sense control loop
```

Reply:
There's possibly also a joke to be made here about real-time guarantees